project-base/storefront/components/Pages/Registration/RegistrationContent.tsx

```diff
@@ -47,61 +47,48 @@ export const RegistrationContent: FC = () => {
 
-                    <div className="mb-10">
                         <RegistrationPassword />
-                    </div>
 
-                    <div className="mb-10">
                         <RegistrationAddress />
-                    </div>
 
-                    <CheckboxControlled
-                        control={formProviderMethods.control}
-                        formName={formMeta.formName}
-                        name={formMeta.fields.gdprAgreement.name}
-                        render={(checkbox) => <ChoiceFormLine>{checkbox}</ChoiceFormLine>}
-                        checkboxProps={{
-                            label: formMeta.fields.gdprAgreement.label,
-                        }}
-                    />
-                    <CheckboxControlled
-                        control={formProviderMethods.control}
-                        formName={formMeta.formName}
-                        name={formMeta.fields.newsletterSubscription.name}
-                        render={(checkbox) => <ChoiceFormLine>{checkbox}</ChoiceFormLine>}
-                        checkboxProps={{
-                            label: formMeta.fields.newsletterSubscription.label,
-                        }}
-                    />
-                    <div className="mt-8 flex w-full justify-center">
-                        <SubmitButton tid={TIDs.registration_submit_button}>{t('Sign up')}</SubmitButton>
-                    </div>
+                        <FormBlockWrapper>
+                            <CheckboxControlled
+                                control={formProviderMethods.control}
+                                formName={formMeta.formName}
+                                name={formMeta.fields.gdprAgreement.name}
+                                render={(checkbox) => <ChoiceFormLine>{checkbox}</ChoiceFormLine>}
+                                checkboxProps={{
+                                    label: formMeta.fields.gdprAgreement.label,
+                                }}
+                            />
+                            <CheckboxControlled
+                                control={formProviderMethods.control}
+                                formName={formMeta.formName}
+                                name={formMeta.fields.newsletterSubscription.name}
+                                render={(checkbox) => <ChoiceFormLine>{checkbox}</ChoiceFormLine>}
+                                checkboxProps={{
+                                    label: formMeta.fields.newsletterSubscription.label,
+                                }}
+                            />
+                            <FormButtonWrapper>
+                                <SubmitButton tid={TIDs.registration_submit_button}>{t('Sign up')}</SubmitButton>
+                            </FormButtonWrapper>
+                        </FormBlockWrapper>
+                    </FormWrapper>
                 </Form>
             </FormProvider>
-        </SimpleLayout>
```

Comment:
Do we need `SimpleLayout` for anything?

Reply:
There still some pages with this layout, I will remove it when I will be styling stores page, because it will be the last which needs it